Cubical/Data/Nat/Base.agda

```diff
@@ -7,6 +7,16 @@ open import Agda.Builtin.Nat public
   using (zero; suc; _+_; _*_)
   renaming (Nat to ℕ)
 
+open import Agda.Builtin.FromNat public
```

Comment:
What is this?

Reply:
Ok, maybe you can add a comment about this so that we don't forget what the instance arguments are there for?